src/utils/jwt.ts

```diff
@@ -186,6 +195,53 @@ export class JWT {
     return jwt.sign(opts.params, keySecret, tokenOptions);
   }
 
+  /**
+   * Creates a new token to be used with a signed statistics request
+   * @param {string} Id - The ID of the object that you'd like to generate a token for
+   * @param {Object} options - Configuration options to use when creating the token
+   * @param {string} [options.keyId] - The signing key ID to use. If not specified, process.env.MUX_SIGNING_KEY is attempted
+   * @param {string} [options.keySecret] - The signing key secret. If not specified, process.env.MUX_PRIVATE_KEY is used.
+   * @param {string} [options.type=video] - Type of token this will be. Valid types are `video`, `asset`, `playback`, or `live_stream`
+   * @param {string} [options.expiration=7d] - Length of time for the token to be valid.
+   * @param {Object} [options.params] - Any additional query params you'd use with a public url. For example, with a thumbnail this would be values such as `time`.
+   * @returns {string} - Returns a token to be used with a signed URL.
+   *
+   * @example
+   * const Mux = require('@mux/mux-node');
+   *
+   * const token = Mux.JWT.sign('some-id', { type: 'video', keyId: 'your key id', keySecret: 'your key secret' });
+   * // Now you can use the token in a url: `https://stream.mux.com/some-playback-id.m3u8?token=${token}`
```

Comment:
This should definitely be updated to be the stat count url

Reply:
Fixed.